js/controller.js

```diff
@@ -290,6 +291,7 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
           'customerUi',
           _.bind(this.onPlayheadTimeChanged, this)
         );
+        this.mb.subscribe(OO.EVENTS.SEEK, 'customerUi', this.onSeek.bind(this));
```

Comment:
You forgot to unsubscribe for this event :)

Reply:
yes, thank you)